### PR TITLE
export alerts to system

### PR DESCRIPTION
**Pull request by mahirchavda**

### CODE REVIEW

File: TA-cyences/metadata/default.meta [hunk 1]
```diff
@@ -31,6 +31,15 @@ export = system
 export = system
 
 
+### ALERT ACTIONS
+
+[alert_actions]
+export = system
+
+[alerts]
```
Comment (VatsalJagani) [inline TA-cyences/metadata/default.meta:39]: Why alerts?

File: TA-cyences/metadata/default.meta [hunk 2]
```diff
@@ -31,6 +31,15 @@ export = system
 export = system
 
 
+### ALERT ACTIONS
+
+[alert_actions]
+export = system
```
Comment (VatsalJagani) [inline TA-cyences/metadata/default.meta:37]: Have we tested below:
When we open Cyences' alert from other App and make changes to alert action parameters, will it update the parameter under etc/apps/cyences_app_for_splunk/local/savedsearches.conf?

File: TA-cyences/metadata/default.meta (hunk 1 above)
Comment (mahirchavda) [reply TA-cyences/metadata/default.meta:39]: Alert_actions to show the alerts action and alerts to show the alert form/html. (without alerts it won't show any fields under the alert action)

File: TA-cyences/metadata/default.meta (hunk 2 above)
Comment (mahirchavda) [reply TA-cyences/metadata/default.meta:37]: yes. it is working as expected

**Review by VatsalJagani — APPROVED** [review]

**Pull request merged**